packages/nextra/styles/code-block.css

```diff
@@ -33,6 +33,10 @@ pre {
     }
   }
 
+  &:not([data-theme]) {
+    @apply nx-px-4;
+  }
```

Comment:
fix broken styles with `codeHighlight: false`
<img width="921" alt="image" src="https://user-images.githubusercontent.com/7361780/212416382-ed1e5a7f-9feb-413f-97ba-b5f93b892606.png">

Reply:
also, I made this PR https://github.com/shuding/nextra/pull/952 works here because `codeHighlight: false` doesn't wrap `<pre/>` with `<div data-rehype-pretty-code-fragment />`